scripts/start/run.R

```diff
@@ -145,86 +145,19 @@ run <- function(start_subsequent_runs = TRUE) {
-        message("  non_optimal.gdx exists, because iteration ", max_iter_no, " did not find a locally optimal solution. ",
-          "modelstat: ", modelstat_no, if (modelstat_no %in% names(explain_modelstat)) paste0(" (", explain_modelstat[modelstat_no], ")"))
-        modelstat[[as.character(max_iter_no)]] <- modelstat_no
-      }
-      if(! file.exists("fulldata.gdx")) {
-        message("! fulldata.gdx does not exist, so output generation will fail.")
-        if (cfg$action == "ce") {
-          stoprun <- TRUE
-        }
-      } else {
-        modelstat_fd <- as.numeric(readGDX(gdx = "fulldata.gdx", "o_modelstat", format = "simplest"))
-        max_iter_fd  <- as.numeric(readGDX(gdx = "fulldata.gdx", "o_iterationNumber", format = "simplest"))
-        message("  fulldata.gdx exists, because iteration ", max_iter_fd, " was successful. ",
-          "modelstat: ", modelstat_fd, if (modelstat_fd %in% names(explain_modelstat)) paste0(" (", explain_modelstat[modelstat_fd], ")"))
-        modelstat[[as.character(max_iter_fd)]] <- modelstat_fd
-      }
-      if (length(modelstat) > 0) {
-        modelstat <- modelstat[which.max(names(modelstat))]
-        message("  Modelstat after ", as.numeric(names(modelstat)), " iterations: ", modelstat,
-                if (modelstat %in% names(explain_modelstat)) paste0(" (", explain_modelstat[modelstat], ")"))
-      }
-      logStatus <- grep("*** Status", readLines("full.log"), fixed = TRUE, value = TRUE)
-      message("  full.log states: ", paste(logStatus, collapse = ", "))
-      if (! all("*** Status: Normal completion" == logStatus)) stoprun <- TRUE
-    }
-  }
+  #====================== Model summary ===========================
 
-  if (identical(cfg$gms$optimization, "nash") && file.exists("full.lst") && cfg$action == "ce") {
-    message("\nInfeasibilities extracted from full.lst with nashstat -F:")
-    command <- paste(
-      "li=$(nashstat -F | wc -l); cat",   # li-1 = #infes
-      "<(if (($li < 2)); then echo no infeasibilities found; fi)",
-      "<(if (($li > 1)); then nashstat -F | head -n 2 | sed -r 's/\\x1B\\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]//g'; fi)",
-      "<(if (($li > 4)); then echo ... $(($li - 3)) infeasibilities omitted, show all with 'nashstat -a' ...; fi)",
-      "<(if (($li > 2)); then nashstat -F | tail -n 1 | sed -r 's/\\x1B\\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]//g'; fi)",
-      "<(if (($li > 3)); then echo If infeasibilities appear some iterations before GAMS failed, check 'nashstat -a' carefully.; fi)",
-      "<(if (($li > 3)); then echo The error that stopped GAMS is probably not the actual reason to fail.; fi)")
-    nashstatres <- try(system2("/bin/bash", args = c("-c", shQuote(command))))
-    if (nashstatres != 0) message("nashstat not found, search for p80_repy in full.lst yourself.")
-  }
-  message("")
+  modelSumData <- modelSummary(".", gams_runtime)
```

Comment:
could we use here "modelSummaryData" instead of "modelSumData"? "Sum" sounds like "summation" for me :-)

Reply:
[Done](https://github.com/remindmodel/remind/compare/dfb06130916c15c300446ebda6f041647f230544..50ff6f730abaa29591a212400ab3dfe87a52e377)